### PR TITLE
mplexporter...

### PR DESCRIPTION
**Pull request by ocefpaf**
... mpld3 and mplleaflet.

### CODE REVIEW

File: recipes/mpld3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        - mpld3
+
+about:
+    home: http://mpld3.github.com
+    license: BSD-3-Clause
+    summary: D3 Viewer for Matplotlib
+
+extra:
+    recipe-maintainers:
+        - jakevdp
```
Comment (ocefpaf) [inline recipes/mpld3/meta.yaml]: @jakevdp is it OK if I add you as a maintainer here? Are there more people you would like to add?

File: recipes/mplleaflet/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        - mplleaflet
+
+about:
+    home: https://github.com/jwass/mplleaflet
+    license: BSD-3-Clause
+    summary: Convert Matplotlib plots into Leaflet web maps
+
+extra:
+    recipe-maintainers:
+        - jwass
```
Comment (ocefpaf) [inline recipes/mplleaflet/meta.yaml]: @jwass I added you as a maintainer here. Let me know if that is OK or if I should remove your GitHub handle from the recipe. (In conda-forge that means you'll have merge rights to the recipe repository.)

**Comment by jakevdp** [comment]
We haven't done an mpld3 release in WAY too long... it's proably due for another one. Every time I've tried, though, I've gotten bogged down in difficulties of simultaneous JS & python release, lack of unit tests for interaction, etc.

**Comment by ocefpaf** [comment]
> We haven't done an mpld3 release in WAY too long

Hehe. I know. I bugged you at SciPy last year and I plan to do the same at PyCon this year!

Well... This version is stable and useful. Let's get that in conda-forge!

**Comment by jakevdp** [comment]
Also the growing list of issues that will never be fixed, because mpld3 is essentially just a hack that out-grew its britches.

**Comment by ocefpaf** [comment]
> Also the growing list of issues that will never be fixed, because mpld3 is essentially just a hack that out-grew its britches.

I do like `mpld3` simplicity, but I understand that it is hard for you to support that library. To be honest I have a few projects that uses `mpld3` (hence this conda package), but I am slowly migrating them to `bokeh` :smile:

**Comment by pelson** [comment]
What's not to love about these recipes. :+1:

**Comment by jakirkham** [comment]
Yeah, `mpld3` is pretty neat indeed. Nice to have it here. :smile:

**Pull request merged**